lib/workers/repository/init/__snapshots__/vulnerability.spec.ts.snap

```diff
@@ -120,7 +85,7 @@ exports[`workers/repository/init/vulnerability detectVulnerabilityAlerts() retur
       "schedule": [],
     },
     "isVulnerabilityAlert": true,
-    "matchCurrentVersion": "== 1.6.7",
+    "matchCurrentVersion": "< 2.2.1.0",
```

Comment:
this seems like a wrong change, this operator is not compatible with pep versioning

Reply:
```js
    it('case test match', () => {
      const result = matcher.matches(
        {
          versioning: 'pep440',
          currentValue: '1.6.2',
        },
        {
          matchCurrentVersion: '<2.2.1.0',
        },
      );
      expect(result).toBeTrue();
    });
```
    This passes

Reply:
https://github.com/renovatebot/pep440/blob/55b074f156ef4a5e261e54792307d0b2b532da1f/lib/operator.js#L12

It is in the list of operators and I tested the pep440 versioning's `matches` method. And it returned true for `pep440.matches( `1.6.2`, `<2.2.1.0`)`

Reply:
wonder why the version changes? Did you changed them in tests?

Reply:
It's because the logic for combining alerts has changed. I haved explained it under `Refactor Old Logic` in the [PR description](https://github.com/renovatebot/renovate/pull/27378#discussion_r1590658120)

Reply:
I've confirmed that unfortunately the REST API doesn't return the exact affected version (e.g. ==1.6.7) and instead returns the affected range (e.g. <2.2.1.0). The fix recommendation (e.g. update to 2.2.1.0) should remain the same. Tested npm and maven in a repo.